packages/tenant-process/test/utils.ts

```diff
@@ -75,6 +76,7 @@ export const getMockTenant = (): Tenant => ({
   createdAt: new Date(),
   attributes: [],
   selfcareId: generateId(),
+  onboardedAt: new Date(),
```

Comment:
As discussed with @Carminepo2  and @galales, this field within the model had to become mandatory, I preferred to leave it optional because then the optionality would conflict with the protobuf V1 decoder, I think it is appropriate to talk about it together and if necessary carry out the changes in another PR.
What do you think?